packages/fleather/test/widgets/editor_toolbar_test.dart

```diff
@@ -0,0 +1,79 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:fleather/fleather.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:quill_delta/quill_delta.dart';
+
+// Workaround to avoid icon and text to generated pixel overflow
+Future<void> loadRobotoFont() async {
```

Comment:
I've removed this function and all the font files without any issues with running tests. What is the purpose of these font related additions?

Reply:
Funny thing. I had the problem when designing the tests, not anymore. Removing